crates/core/src/subscription/execution_unit.rs

```diff
@@ -278,6 +277,23 @@ impl ExecutionUnit {
         }))
     }
 
+    fn eval_query_expr_against_memtable<'a>(
+        ctx: &'a ExecutionContext,
+        db: &'a RelationalDB,
+        tx: &'a TxMode,
+        mem_table: MemTable,
+        eval_incr_plan: &'a QueryExpr,
+        _replaced_source_id: SourceId,
```

Comment:
Can we remove this now?

Reply:
What do you want to remove? This whole method?

Reply:
`_replaced_source_id`

Reply:
Sure, done.